ezyquant/reader.py

```diff
@@ -21,17 +21,13 @@
 
 
 class SETDataReader:
-    """SETDataReader read PSIMS data."""
 
-    def __init__(self, sqlite_path: str):
-        """SETDataReader constructor.
+    _sqlite_path: Optional[str] = None
 
-        Parameters
-        ----------
-        sqlite_path : str
-            path to sqlite file e.g. /path/to/sqlite.db
-        """
-        self._sqlite_path = sqlite_path
+    def __init__(self):
+        """SETDataReader read PSIMS data."""
+        if self._sqlite_path == None:
+            raise InputError("sqlite_path is not set")
```

Comment:
แก้ message เป็น "You need to connect sqlite using ezyquant.connect_sqlite(sqllite_path)."

Reply:
แก้แล้ว